app/views/documents/show/_actions.html.erb

```diff
@@ -12,32 +12,36 @@
     <% elsif @edition.status.removed? %>
       <%= create_edition_button(@edition) %>
     <% elsif @edition.status.submitted_for_review? %>
-      <%= publish_button(@edition) %>
-      <%= preview_button(@edition, secondary: true) %>
-      <%= delete_draft_link(@edition, "app-link--right") %>
+      <% if @edition.schedulable? %>
+        <%= schedule_confirmation_button(@edition) %>
+        <%= preview_button(@edition, secondary: true) %>
+        <%= delete_draft_link(@edition, "app-link--right") %>
+      <% else %>
+        <%= publish_button(@edition) %>
+        <%= preview_button(@edition, secondary: true) %>
+        <%= delete_draft_link(@edition) %>
+        <%= create_schedule_date_link(@edition, "app-link--right") %>
+      <% end %>
     <% elsif !@edition.revision_synced? %>
       <%= create_preview_button(@edition) %>
       <%= delete_draft_link(@edition) %>
     <% elsif @edition.withdrawn? %>
       <%= undo_withdraw_button(@edition) %>
     <% elsif @edition.scheduled? %>
       <%= preview_button(@edition) %>
-      <%= unschedule_button(@edition) %>
-      <%= delete_draft_link(@edition) %>
     <% else %>
       <%= submit_for_2i_button(@edition) %>
       <%= preview_button(@edition, secondary: true) %>
-      <% if @edition.schedulable? && current_user.has_permission?(User::PRE_RELEASE_FEATURES_PERMISSION) %>
-        <%= schedule_link(@edition) %>
-      <% else %>
-        <%= publish_link(@edition) %>
+      <% if @edition.schedulable? %>
+        <%= delete_draft_link(@edition) %>
+        <%= schedule_confirmation_link(@edition, "app-link--right") %>
+      <% elsif @edition.editable? && current_user.has_permission?(User::PRE_RELEASE_FEATURES_PERMISSION) %>
+        <p class="govuk-body govuk-!-margin-bottom-3">
+          <%= publish_link(@edition) %>
+          <%= create_schedule_date_link(@edition, "app-link--right") %>
+        </p>
+        <%= delete_draft_link(@edition) %>
```

Comment:
I feel like we should have a schedule link when revision hasn't synced too - since that can be used to set a date even if we can't write to Publishing API

Reply:
Oh and should remember to hide that from users without permission

Reply:
Done